core/src/main/scala/kafka/cluster/Replica.scala

```diff
@@ -56,6 +58,8 @@ class Replica(val brokerId: Int, val topicPartition: TopicPartition) extends Log
 
   def lastSentHighWatermark: Long = _lastSentHighWatermark
 
+  def lastReassignmentLag: Long = _lastReassignmentLag
```

Comment:
nit: I think `reassignmentLag` is accurate here. I'm not a native english speaker but I believe `last` may sound weird here

Reply:
Ok, will update it.